frappe/desk/doctype/system_console/system_console.py

```diff
@@ -26,7 +26,7 @@ def run(self):
 		else:
 			frappe.db.rollback()
 
-		frappe.get_doc(dict(doctype="Console Log", script=self.console, output=self.output)).insert()
+		frappe.get_doc(dict(doctype="Console Log", script=self.console)).insert()
```

Comment:
Better option would be to store only first few character/lines and add `...` after it?


It does sometimes help to see the log output to find out what the user retrieved using system console.

Reply:
User can run the same script and check output in the system console

Reply:
Sometimes we need to see the output of that exact time, not now. But it is really an edge case and should be handled seperately.